packages/skeleton-common/src/classes/steps.ts

```diff
@@ -0,0 +1,19 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes.js' with { type: 'macro' };
+
+export const classesSteps = defineSkeletonClasses({
```

Comment:
Note there's a design issue here where the last item is taking up the same width as the the prior items, even without the separator. This results in a wide gap on the left-side.

<img width="1705" height="362" alt="Screenshot 2025-11-13 at 11 08 24 AM" src="https://github.com/user-attachments/assets/f3972b7f-0ec9-4a15-aa34-19e8a7884fe7" />

What we need is something more like this:

<img width="833" height="198" alt="Screenshot 2025-11-13 at 11 11 09 AM" src="https://github.com/user-attachments/assets/1fbc1b6e-9e5f-4849-a014-eba7e072ea87" />

I'm short on time today, but if you need help determining how to do this I can circle back and help!

I might suggest looking up Zag or Ark's default styles for their examples and convert from that Tailwind utility classes.

Reply:
fixed, using by changing flex behavior on the `last:` item